src/java/org/apache/cassandra/service/accord/serializers/CommandSerializers.java

```diff
@@ -75,8 +75,7 @@ private CommandSerializers()
     public static final IVersionedSerializer<TxnId> nullableTxnId = NullableSerializer.wrap(txnId);
     public static final TimestampSerializer<Timestamp> timestamp = new TimestampSerializer<>(Timestamp::fromBits);
     public static final IVersionedSerializer<Timestamp> nullableTimestamp = NullableSerializer.wrap(timestamp);
-    public static final TimestampSerializer<Ballot> ballot = new TimestampSerializer<>(Ballot::fromBits);
-    public static final IVersionedSerializer<Ballot> nullableBallot = NullableSerializer.wrap(ballot);
+    public static final BallotSerializer ballot = new BallotSerializer(); // permits null
```

Comment:
I am a bit confused why we have nullable ballot and ballot if this one permits null?